__tests__/app.test.js

```diff
@@ -3,7 +3,8 @@ const request = require("supertest");
 const db = require("../db/connection");
 const seed = require("../db/seeds/seed");
 const data = require('../db/data/test-data/index')
-const endpoints = require('../endpoints.json')
+const endpoints = require('../endpoints.json');
+const { string } = require('pg-format');
```

Comment:
was this autoimported?

Reply:
i think i merged a few branches code together by accident, tried deleting code from other branches but this probably slipped through